erp/management/commands/export_to_datagouv.py

```diff
@@ -1,12 +1,50 @@
+import requests
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
-from erp.jobs import export_to_datagouv
+from erp.export.export import export_schema_to_csv, upload_to_datagouv
+from erp.export.mappers import EtalabMapper
+from erp.models import Erp
 
 
 class Command(BaseCommand):
+    help = "Exporte vers datagouv"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Afficher les erreurs",
+        )
+
     def handle(self, *args, **options):
-        try:
-            export_to_datagouv.job()
-            print("Data exported successfully to 'export.csv'")
-        except KeyboardInterrupt:
-            print("Interrompu.")
+        verbose = options.get("verbose", False)
+        csv_path = "acceslibre.csv"
```

Comment:
How about creating the export file in a tempfile? https://docs.python.org/3.8/library/tempfile.html

Reply:
The problem is that when you upload a file, the ressource title is changed to the file name (same with the website). And I can't name a tmp with the tempfile module (yeah NamedTemporaryFile doesn't mean you can choose a name)

Reply:
Maybe you can create a tmp directory and create the file in it? 🤔